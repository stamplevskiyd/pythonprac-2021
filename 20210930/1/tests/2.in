1,2,3,4,5,0,10,100,1000
