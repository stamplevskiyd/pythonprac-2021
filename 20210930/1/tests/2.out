[0, 10, 100, 1000, 1, 2, 3, 4, 5]
